floodviz/static/js/hydrograph.js

```diff
@@ -143,39 +141,43 @@
 				update(data);
 			});
 		};
-    
+
 		/**
-		 * Handle all mouse over movements for calling element. 
+		 * Displays tooltip for hydrograph at a datapoint in addition to
+		 * corresponding map site tooltip
 		 */
-		self.mouseover = function(d) {
+		self.series_tooltip_show = function(d) {
 			d3.select(d.data.name).classed("gage--hover", true);
 			focus.attr("transform", "translate(" + x(d.data.time_mili) + "," + y(d.data.value) + ")");
 			focus.select("text").html(d.data.key + ": " + d.data.value + " cfs " + " " + d.data.time + " " + d.data.timezone);
-			// Interative linking with map
-			FV.map_figure.mousemove(d.data.name, d.data.key);
+			// Interactive linking with map
+			FV.map_figure.site_tooltip_show(d.data.name, d.data.key);
```

Comment:
Wasn't quite sure why you think will interfere. The map figure can call a hydromodule public function that highlights itself.